crates/rpc/src/lib.rs

```diff
@@ -266,6 +266,32 @@ impl Handler {
         Ok(sender.finish().await?)
     }
 
+    pub async fn get_code(address: Address, chain_id: u32) -> jsonrpc_core::Result<Option<String>> {
```

Comment:
this function isn't well placed here. this handler is for implementing the JSON-RPC server from ethereum. you're wrapping a rpc endpoint but via a custom function that's not compatible with the server itself

easiest fix is to just move this function to a utils.rs module